parsec/core/cli/reencrypt_workspace.py

```diff
@@ -0,0 +1,46 @@
+# Parsec Cloud (https://parsec.cloud) Copyright (c) AGPLv3 2016-2021 Scille SAS
+
+import click
+
+
+from parsec.utils import trio_run
+from parsec.api.data import EntryName
+from parsec.cli_utils import cli_exception_handler, spinner
+from parsec.core import logged_core_factory
+from parsec.core.types import LocalDevice
+from parsec.core.config import CoreConfig
+from parsec.core.cli.utils import cli_command_base_options, core_config_and_device_options
+
+
+async def _reencrypt_workspace(config: CoreConfig, device: LocalDevice, name: EntryName) -> None:
+    async with logged_core_factory(config, device) as core:
+        workspace = core.find_workspace_from_name(name)
+        workspace_id = workspace.id
+        workspace_fs = core.user_fs.get_workspace(workspace_id)
+        reenc_needs = await workspace_fs.get_reencryption_need()
+        if not reenc_needs.need_reencryption:
+            raise RuntimeError("The workspace does not need to be reencrypted")
```

Comment:
Raising an error for this seems a bit to harsh, instead I'd go with a nice `click.echo` message then return (so status code of the process would be 0)

Reply:
Done.